tests/lang_tests_common.rs

```diff
@@ -7,7 +7,7 @@ use std::{
 use lang_tester::LangTester;
 use tempfile::TempDir;
 
-fn main() {
+pub fn main_inner(modify_command: fn(&mut Command) -> &mut Command) {
```

Comment:
I'd prefer to use a config argument instead of a function. That argument can be a simple enum with `Debug/Release` variants for now. We could convert that later to a struct if needed.